doc/apiv3-documentation.apib

```diff
@@ -4180,151 +4752,510 @@ A project link must be set when creating work packages through this route.
+                "message": "You are not allowed to create a relation."
             }
 
-+ Response 404 (application/hal+json)
++ Response 409 (application/hal+json)
 
-    Returned if the project does not exist or the client does not have sufficient permissions to see it.
+    Returned if there already exists a relation between the given work packages of that type
+    or if the relation is not allowed.
 
-    **Required permission:** view project
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:UpdateConflict",
+                "message": "A relation of that type between the work packages either already exists or is not allowed."
+            }
+
++ Response 422 (application/hal+json)
+
+    Returned if:
+
+    * the client tries to write a read-only property (`PropertyIsReadOnly`)
+    * a constraint for a property was violated (`PropertyConstraintViolation`)
+    * the client provides a link to an invalid resource (`ResourceTypeMismatch`)
 
     + Body
 
             {
                 "_type": "Error",
-                "errorIdentifier": "urn:openproject-org:api:v3:errors:NotFound",
-                "message": "The specified project does not exist."
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:PropertyConstraintViolation",
+                "message": "Delay must be a number greater than or equal to 0",
+                "_embedded": {
+                    "details": {
+                        "attribute": "delay"
+                    }
+                }
             }
 
-## Create Work Package [POST]
+## List relations [GET]
 
-When calling this endpoint the client provides a single object, containing at least the properties and links that are required, in the body.
-The required fields of a WorkPackage can be found in its schema, which is embedded in the respective form.
-Note that it is only allowed to provide properties or links supporting the write operation.
+Lists all relations this work package is involved in.
 
 + Parameters
-    + id (required, integer, `1`) ... Project id
+    + work_package_id (required, integer, `1`) ... Work package id
 
-    + notify = `true` (optional, boolean, `false`) ... Indicates whether change notifications (e.g. via E-Mail) should be sent.
-    Note that this controls notifications for all users interested in changes to the work package (e.g. watchers, author and assignee),
-    not just the current user.
++ Response 302 (text/plain)
 
-+ Response 200 (application/hal+json)
+  + Headers
 
-    [Work Package][]
+            Location: /api/v3/relations?involved={work_package_id}
 
-+ Response 400 (application/hal+json)
+  + Body
 
-    Occurs when the client did not send a valid JSON object in the request body.
+            You are being redirected to /api/v3/relations?involved={work_package_id}
+
+# Work package relation schema [/api/v3/work_packages/{id}/relations/schema]
```

Comment:
If we leave out the available values from the generic schema and embed them in the form, we should be able to remove the work package specific relation schemas.